server/src/providers/blockchain.ts

```diff
@@ -270,17 +282,25 @@ export class blockchain {
       }
     }
 
+    log.debug('updateUBIQuota - events data parsed:', {
```

Comment:
it will be a large data pack.
1. probably better to lo log { date, ubi_quata } pairs
2. what's happening in AboutClaimTransactionProvider.updateOrSet ? did we trying to save all this data block ? or we performing some aggregations ? we need to check/log it also

Reply:
1. actually this data is n need to log - I will change it to print just quantity
2. Its saving/updating the data in db